src/Library/demos/Camera/main.js

```diff
@@ -0,0 +1,76 @@
+import Gtk from "gi://Gtk";
+import Xdp from "gi://Xdp";
+import XdpGtk from "gi://XdpGtk4";
+import Gio from "gi://Gio";
+import GObject from "gi://GObject";
+import GLib from "gi://GLib";
+import Gst from "gi://Gst";
+
+const portal = new Xdp.Portal();
+const parent = XdpGtk.parent_new_gtk(workbench.window);
+const video = workbench.builder.get_object("video");
+
+const button = workbench.builder.get_object("button");
+async function onClicked() {
+  if (portal.is_camera_present) {
+    const pwRemote = await portal.open_pipewire_remote_for_camera();
+    print("Pipewire remote opened for camera");
+    Gst.init(null);
+
+    // Create the pipeline
+    const pipeline = new Gst.Pipeline();
+
+    // Create elements
+    const source = Gst.ElementFactory.make("pipewiresrc", "source");
+    const videoConvert = Gst.ElementFactory.make(
+      "videoconvert",
+      "videoConvert",
+    );
+    const videoSink = Gst.ElementFactory.make("gtksink", "videoSink");
```

Comment:
gtksink is gtk3, i don't think mixing gtk3 and gtk4 in the same process is recommended.